django/contrib/sessions/checks.py

```diff
@@ -0,0 +1,20 @@
+from django.apps.registry import Tags
+from django.conf import settings
+from django.core import checks
+from django.core.checks import register
+
+
+@register(Tags.sessions)
+def check_names_conflict(app_configs=None, **kwargs):
+    errors = []
+
+    if settings.SESSION_COOKIE_NAME == settings.LANGUAGE_COOKIE_NAME:
+        errors.append(
+            checks.Error(
+                "'SESSION_COOKIE_NAME' and 'LANGUAGE_COOKIE_NAME' settings are equal.",
```

Comment:
"The SESSION_COOKIE_NAME and LANGUAGE_COOKIE_NAME settings must be different."
(don't think the hint is needed as it's just repetitive)